### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by rnovikovP**

### CODE REVIEW

**Comment by arvenil** [comment]
@rnovikovP to be honest, maybe we should just drop `master` from all repos. We don't need it but it increases build time.

**Comment by arvenil** [comment]
@rnovikovP @delgod @EvgeniyPatlan  Percona doesn't provide mysql 5.5 as docker image.
```
Pulling mysql (percona/percona-server:5.5)...
ERROR: manifest for percona/percona-server:5.5 not found
```

File: .travis.yml [hunk 1]
```diff
@@ -5,13 +5,18 @@ language: go
 go:
 - 1.9.x
 - 1.10.x
-- master
+
+matrix:
+  allow_failures:
+    - go: master
```
Comment (arvenil) [inline .travis.yml]: @rnovikovP Then also drop this whole matrix

File: .travis.yml [hunk 2]
```diff
@@ -5,13 +5,18 @@ language: go
...
 
 env:
 - MYSQL_IMAGE=mysql/mysql-server:5.5
 - MYSQL_IMAGE=mysql/mysql-server:5.6
 - MYSQL_IMAGE=mysql/mysql-server:5.7
 - MYSQL_IMAGE=mysql/mysql-server:8.0
+- MYSQL_IMAGE=percona/percona-server:5.5
```
Comment (arvenil) [inline .travis.yml]: @rnovikovP We don't have percona-server:5.5 so just drop it too.

**Comment by rnovikovP** [comment]
@arvenil  ..maybe MariaDB should be added?

**Pull request merged**